src/services/EcoBridge/store/EcoBridge.selectors.ts

```diff
@@ -117,6 +119,7 @@ export const selectBridgeListsLoadingStatus = createSelector(
     (state: AppState) => state.ecoBridge['arbitrum:testnet'].listsStatus,
     (state: AppState) => state.ecoBridge['arbitrum:mainnet'].listsStatus,
     (state: AppState) => state.ecoBridge['socket'].listsStatus,
+    (state: AppState) => state.ecoBridge['connext'].listsStatus,
```

Comment:
it would come in handy here also...

Reply:
https://github.com/levelkdev/swapr-dapp/pull/1008#discussion_r90150709